scripts/codemods/ac3-to-ac4/src/renames.ts

```diff
@@ -0,0 +1,909 @@
+    { from: "VariablesOption" },
+  ].map(
+    moveInto({
+      from: { module: "@apollo/client/utilities" },
+      to: { module: "@apollo/client/utilities/internal" },
+      importType: "type",
+    })
+  ),
+  ...[{ from: "VariablesOption" }, { from: "NoInfer" }].map(
+    moveInto({
+      from: { module: "@apollo/client/react" },
+      to: { module: "@apollo/client/utilities/internal" },
+      importType: "type",
+    })
+  ),
+  ...[{ from: "__DEV__" }, { from: "DEV", to: "__DEV__" }].map(
+    moveInto({
+      from: {
+        module: "@apollo/client/utilities/global",
+        alternativeModules: ["@apollo/client/utilities"],
+      },
+      to: { module: "@apollo/client/utilities/environment" },
+      importType: "value",
+    })
+  ),
+  ...[{ from: "maybe" }, { from: "global" }].map(
+    moveInto({
+      from: {
+        module: "@apollo/client/utilities/global",
+        alternativeModules: ["@apollo/client/utilities"],
+      },
+      to: { module: "@apollo/client/utilities/internal/globals" },
+      importType: "value",
+    })
+  ),
+  ...[
+    { from: "invariant" },
+    { from: "newInvariantError" },
+    { from: "InvariantError" },
+  ].map(
+    moveInto({
+      from: { module: "@apollo/client/utilities/global" },
+      to: { module: "@apollo/client/utilities/invariant" },
+      importType: "value",
+    })
+  ),
+  ...[
+    // move into `ApolloClient` type namespace
+    { from: "ApolloClientOptions", to: "Options" },
+    { from: "DefaultOptions" },
+    { from: "DevtoolsOptions" },
+    { from: "MutateResult" },
+    { from: "MutationOptions", to: "MutateOptions" },
+    {
+      from: "QueryOptions",
+      postProcess: reorderGenerics([1, 0]),
+    },
+    { from: "RefetchQueriesOptions" },
+    { from: "RefetchQueriesResult" },
+    {
+      from: "SubscriptionOptions",
+      to: "SubscribeOptions",
+      postProcess: reorderGenerics([1, 0]),
+    },
+    { from: "WatchQueryOptions", postProcess: reorderGenerics([1, 0]) },
+  ].map(
+    moveInto({
+      from: { module: "@apollo/client" },
+      to: { namespace: "ApolloClient" },
+      importType: "type",
+    })
+  ),
+  ...[
+    // move into `ObservableQuery` type namespace
+    { from: "ApolloQueryResult", to: "Result" },
+    { from: "SubscribeToMoreOptions" },
+    { from: "FetchMoreOptions" },
+  ].map(
+    moveInto({
+      from: { module: "@apollo/client" },
+      to: { namespace: "ObservableQuery" },
+      importType: "type",
+    })
+  ),
+  ...[
+    // move into `ApolloCache` type namespace
+    { from: "WatchFragmentOptions" },
+    { from: "WatchFragmentResult" },
+  ].map(
+    moveInto({
+      from: { module: "@apollo/client/cache" },
+      to: { namespace: "ApolloCache" },
+      importType: "type",
+    })
+  ),
+  ...[{ from: "Context", to: "DefaultContext" }].map(
+    moveInto({
+      from: {
+        module: "@apollo/client/react",
+      },
+      to: { namespace: "@apollo/client" },
```

Comment:
```suggestion
      to: { module: "@apollo/client" },
```

Does this need to be `module`?